### DISCUSSION TITLE
New Discord and Matrix just dropped

### DISCUSSION THREAD

**Discussion by Ottatop** [pinnacle-comp/pinnacle#185] [Announcements]
Yes, I'm going to keep using that line

Hey Discord Degens and Matrix Messengers,

You asked (and by you I mean one person) and we (by we I also mean one person) delivered! We've now got a [Discord server](https://discord.gg/JhpKtU2aMA) and [Matrix space](https://matrix.to/#/#pinnacle:matrix.org) to talk about this little Wayland compositor I'm writing. Most Discord channels and Matrix rooms are bridged so you can see all the stuff the other side is talking about. Hopefully I set everything up correctly :crossed_fingers:, so if you can't join or I screwed up the permissions yell at me under this post. See ya there!

**Comment by ildar** [comment]
thanks for doing that. With that I fell that Pinnacle has the great
potential to grow.
And it's so advanced to set up the bridge right from the start.

But I think permissions on the Matrix side aren't fully correct. I could
join the space and the rooms only by link. Search didn't show Pinnacle.

**Reply by Ottatop** [reply]
Weird, I can't seem to expose any of the rooms to matrix.org for searching. Gonna have to look into it

**Comment by ildar** [comment]
May it be that the search indices aren't updated? not sure how it really
works though

**Reply by thmasq** [reply]
This is my guess as I couldn't confirm it, but I think the matrix.org instance might have full text search disabled. It's computationally heavy at their scale, and you usually have to set a single language to properly index text
